Articles/Blog/what-is-an-inclusive-minute.md

```diff
@@ -0,0 +1,143 @@
+In some of the worst situations, it may be necessary to seek the help of human resources or an ombuds program, if available.
+If your institution is committed to its stated values, it should not be hard to find higher level managers willing to back you up.
+
+#### We already invest heavily in inclusion...in our code
+
+While a focus on inclusive practices may seem like a new idea, the truth is the code we've been writing has for decades exemplified all the best goals of inclusion.
+For example, our community desires *interoperable* libraries so that, for example, an application can easily swap one solver library (e.g. PETSc) for another (e.g. HYPRE).
+Interoperability is a manifestation of inclusion in the code we write.
+
+We also value both API (e.g. compile-time) and ABI (link-time) compatibility in successive or related versions of the same library.
+This is yet another manifestation of inclusion in the code we write.
+Likewise, we want our applications used by people all over the world and as such, value having interfaces (e.g. GUIs, CLIs) as well as documentation available in multiple human languages.
+For example, the VisIt application GUI supports English, French and Spanish and is designed to easily support other human languages.
+
+We also desire useful libraries to support multiple different programming language interface bindings such as C, C++, Fortran, Python or Java.
+For example, the Conduit library can be used *naturally* within C, C++, Fortran and Python programs.
+
+The whole goal of performance portability is to ensure HPC/CSE applications can operate efficiently on a wide variety of computational resources (e.g. CPUs, GPUs, co-processors, FPGAs, etc.).
+Performance portability is an extremely challenging design goal requiring substantial effort and resources to achieve.
+It and the effort we invest in it is yet another example of how the goals of inclusion (e.g. supporting a wide variety of computational resources) manifest in the actual code we design and write.
+
+We value computational kernels and tools that operate on single, double, quad and mixed or arbitrary precision.
+For example, the ZFP compression library supports single, double precision and support for half and quad precision is on the todo list.
+
+The truth is, a substantial level of effort in the design, implementation and support of the software our HPC/CSE community develops and maintains is all about inclusion (of other software).
+It only makes sense that we would be equally committed, if not more so, to inclusion of other people as well.
+Commercial organizations whose business depends on reach and growth (e.g. [Google](https://about.google/belonging/at-work/), [Microsoft](https://inclusive.microsoft.design), [IBM](https://www.ibm.com/blogs/corporate-social-responsibility/2022/04/diversity-equity-inclusion/) and [NVIDIA](https://www.nvidia.com/en-us/about-nvidia/careers/diversity-and-inclusion/)) are fully committed to inclusion. 
```

Comment:
I think this line would fit better somewhere else. The sentence before is a great ending to the article, and this sentence feels somewhat "tacked on." It might make sense earlier (e.g., in a "What is inclusion" section).

Reply:
Agreed...will take care of that shortly as well.

Reply:
Addressed in eada8259e19a5c45b08f4a591f4398a9d5a472fb